src/components/Books/Book/Progress/Progress.css

```diff
@@ -0,0 +1,35 @@
+.progress-container {
```

Comment:
The progress bar circles do not match the design provided. The circle border is filled with the `#307bbe` color in the design.
Kindly follow the design guidelines.
![circular](https://user-images.githubusercontent.com/35504149/162277082-0d5c2f0f-e34b-4c1b-b9e1-dc8d487e5eb9.png)